src/Illuminate/Http/Client/PendingRequest.php

```diff
@@ -108,10 +117,10 @@ class PendingRequest
     /**
      * Create a new HTTP Client instance.
      *
-     * @param  \Illuminate\Http\Client\Factory|null  $factory
```

Comment:
Please don't delete these spaces, and make sure your other param doc also has 2 spaces either side of the type.